src/modules/Dashboard/components/ParticipantSnippet/ParticipantSnippet.styles.ts

```diff
@@ -0,0 +1,7 @@
+import { styled } from '@mui/material';
+
+import { StyledBodyLarge, ellipsisTextCss } from 'shared/styles';
+
+export const StyledText = styled(StyledBodyLarge)`
+  ${ellipsisTextCss}
+`;
```

Comment:
I'm not 100% certain how this compiles, but as `ellpsisTextCss` is already the output of a `css` function, it may be better to write this as:

```suggestion
export const StyledText = styled(StyledBodyLarge)(ellipsisTextCss);
```

… to reuse the existing set of styles, rather than creating a new template literal to be parsed, consisting solely of the embedded styles. This goes for the same usage [here](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1706/files#diff-a55c2a9f8054d4d50378b3b63c84b25534a823578581f165d43c7616572fce8cR152-R155).

Reply:
Sweet, thanks! I didn't realize this type of string literal syntax was just passing a string as an argument to a callback. 🤦🏻‍♂️